bool LeftScannerContaminationWarning
bool LeftScannerContaminationError
bool LeftScannerAppError
bool LeftScannerDeviceError
bool LeftScannerEmergencyZoneStatus
bool LeftScannerSoftStopZoneStatus
bool LeftScannerReducedSpeedZoneStatus
bool LeftScannerActive
bool RightScannerContaminationWarning
bool RightScannerContaminationError
bool RightScannerAppError
bool RightScannerDeviceError
bool RightScannerEmergencyZoneStatus
bool RightScannerSoftStopZoneStatus
bool RightScannerReducedSpeedZoneStatus
bool RightScannerActive
bool LeftEmergencyStopButtonStatus
bool RightEmergencyStopButtonStatus
bool FlexiCPUStatus
bool SafetyEncoderStatus
bool StandstillStatus
